### PR TITLE
LPS-177264

### PR DESCRIPTION
**Pull request by yuhai**
@tinatian, @dantewang, the pr is based on LPS-170859(https://github.com/liferay-core-infra/liferay-portal/pull/4024). I will rebase it after LPS-170859 is merged to master.

### CODE REVIEW

**Comment by yuhai** [comment]
@tinatian, @dantewang, I just update assertEquals() params, from assertEquals(String, Object, Object) to assertEquals(Object, Object), so no need to trigger ci test again.

**Comment by yuhai** [comment]
ci:test:sf

File: modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java [hunk 1]
```diff
@@ -562,6 +562,49 @@ public void testAddAndDeleteCompanyWithUserGroupAndUserGroupRole()
...
+			String expectedLanguageId = defaultUser.getLanguageId();
+			String expectedTimeZoneId = defaultUser.getTimeZoneId();
+
+			String langaugeId = "en_US";
+
+			if (langaugeId.equals(expectedLanguageId)) {
+				langaugeId = "fr_FR";
+			}
+
+			String timeZoneId = "UTC";
```
Comment (dantewang) [inline modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java:583]: @yuhai It seems UTC is not a timezone, but a time standard. Are you sure you should use this?

File: modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java [hunk 2]
```diff
@@ -562,6 +562,49 @@ public void testAddAndDeleteCompanyWithUserGroupAndUserGroupRole()
...
+
+			if (timeZoneId.equals(expectedTimeZoneId)) {
+				timeZoneId = "GMT";
+			}
+
+			defaultUser.setLanguageId(langaugeId);
```
Comment (dantewang) [inline modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java:589]: @yuhai 
Steps to reproduce:
1. Start portal from empty database with "company.default.locale=en_US"
2. Shutdown portal and check database table User_, check the default user's language is "en_US"
3. Set "company.default.locale=es_ES" and restart portal
4. check database table User_, check the default user's language is "es_ES"

This doesn't reflect the steps as you are updating the default user by yourself in the test?

**Comment by dantewang** [comment]
@yuhai Conflicts, please resend.

**Pull request closed without merging**